doc/crypto/api/ops/ka.rst

```diff
@@ -146,6 +237,8 @@ Standalone key agreement
         The first ``(*output_length)`` bytes of ``output`` contain the raw shared secret.
     .. retval:: PSA_ERROR_INVALID_HANDLE
         ``private_key`` is not a valid key identifier.
+    .. retval:: PSA_ERROR_ALREADY_EXISTS
```

Comment:
This error is in the wrong list, it should be for `psa_key_agreement`.

Reply:
Thanks. Evidence of use of cut and paste... 😀